README.md

```diff
@@ -66,6 +66,16 @@ We rely on different `django` and `mypy` versions:
 | 0.12.x | old semantic analyzer (<0.711), dmypy support | 2.1.x | ^3.6
 
 
+## Plugin settings
+
+An optional strictness flag is available for stronger checks:
+
+- `warn_manager_type_created_in_model_body`
```

Comment:
We don't need to make this a setting. Since this error can be fixed with a simple `# type: ignore` comment.

Reply:
Alright, I'll drop the setting